### PR TITLE
Remote postgresql is not supported

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/23875
https://bugzilla.suse.com/show_bug.cgi?id=1220376

# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
Cosmetic changes such as fixing typos do not need log entries (nevertheless it is important to fix typos, etc.)!
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
* Does this PR need to be backported? If yes, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed):

- master
- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

**Comment by mcalmer** [comment]
@admd @rjmateus Do you remember the RDS work we merged month ago into master? Did we told them that this is "supported" in the next SUMA version?

**Comment by rjmateus** [comment]
@mcalmer If I remember correctly, we decided it would be supported in some future version, but we never come to a decision when we should start supporting that because we where not testing it on a regular basis. BUt Maybe @admd remembers more about it

**Comment by admd** [comment]
That's correct. We didn't announce it because we haven't tested it and decided to support it at some later point. We could announce it after 5.0 with some MU or even with 5.1.

File: modules/installation-and-upgrade/pages/postgresql-requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[[installation-postgresql-requirements]]
+= {postgresql} Requirements
+
+{postgresql} is the only supported database.
+Using a remote {postgresql} database or remote file systems (such as NFS) with the {postgresql} database is not supported.
+In other words, {postgresql} should be on the fastest available storage for {productname}.
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/postgresql-requirements.adoc:6]:
```suggestion
In other words, {postgresql} should be on the fastest available storage for {productname}.
```

I don't quite understand what is meant by this sentence. Can it be somehow rephrased? I'm happy to brainstorm it together.

Comment (keichwa) [reply modules/installation-and-upgrade/pages/postgresql-requirements.adoc:6]: Maybe, we should say "storage device".  There are dedicated storage systems.  If supported, SUMA or Uyuni can make use of them.  That's how I understand it.  If I'm wrong, experts will hopefully propose a better text.

**Comment by keichwa** [comment]
Thanks for all your feedback thus far!  I see that there is still some doubt about the wording--thus I'm turning it back to draft status.

I'd appreciate if you could change my text proposal in a way that it would be suitable for 4.3.  Then I can create a followup PR for 5.0.  I'll try to change the merge target accordingly.

**Review by mcalmer — APPROVED** [review]

**Pull request merged**